posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+ 
+Ippon USA offers a helping hand to those impacted by natural disasters. 
+ 
+“Let us remember: One book, one pen, one child, and one teacher, can change the world. Never doubt that a small group of thoughtful committed citizens can change the world. Indeed, it is the only thing that ever has.” - Malala Yousafzai
+ 
+About Me:
```

Comment:
These subtitles should have a #'s for markdown. ie

# Level 1
## Level 2
### Level 3

I suggest making all your subtitles level 1: [About Me, Background, My Time On Project]